saving color mode

aloofbit opened the pull request:
Hi I wanted to add feature. You change the dark/light mode and when reload the page it stays!

![reload-save-color](https://user-images.githubusercontent.com/58625258/215282226-65faa7bf-1cb1-495a-bcbf-c75fc94f0387.gif)

<img width="546" alt="image" src="https://user-images.githubusercontent.com/58625258/215283097-e8f53a4d-f062-4bea-854e-915f28172be9.png">


## html
- added "made with heart"

## css
- put the themes in variables this made it easier to change
- put a transition on more so it is not so sharp

# overall
- move some files
<img width="386" alt="image" src="https://user-images.githubusercontent.com/58625258/215283416-6084a3f9-f9cd-499e-b839-710a297a4399.png">

- deleted all package.json stuff (probably in trouble for this)
    - no build or serve can drop html in browser now. but put serve in case.
- remove github and put in .nojekyl. can host off main branch :root now
- changed some to make download little less

**Before**: 
<img width="1839" alt="image" src="https://user-images.githubusercontent.com/58625258/215282949-f0c9f6c2-8aea-4e88-89ef-134f44a61af2.png">


**After**: 
<img width="1828" alt="image" src="https://user-images.githubusercontent.com/58625258/215282933-a97e3edc-3446-48a9-93d8-a8323a58b8d2.png">

aloofbit committed:
im in trouble

aloofbit committed:
images

aloofbit committed:
links

aloofbit committed:
match original design

aloofbit committed:
tidy

FichteFoll commented on src/script.js:
```diff
@@ -0,0 +1,17 @@
+var state = localStorage.getItem('toggle') === 'true' ? true : false
```
Please use a more descriptive name like "darkMode" or "style" and then set it to "light" or "dark".

aloofbit replied:
ok i tried to improve on this! please see!

FichteFoll commented on index.html:
```diff
@@ -1,110 +1,94 @@
-    <script src="dist/js/main.min.js"></script>
+         </div>
+      </div>
+      <div class="love-sublime">Made with ❤️ on Sublime Text</div>
```
I find these "made with :heart:" kinda cringe and superfluous tbh, but I'm not strictly against it if @TerminalFi likes it. It just doesn't convey any useful information (like, surely you didn't make it with hate or something, but even if you did you probabaly wouldn't tell).

aloofbit replied:
ok ok i understand! but unfortunately i am only the developer and you are going to have to speak with my product manager :(

<img width="684" alt="image" src="https://user-images.githubusercontent.com/58625258/215297196-7e146e1e-0789-4ad5-81a8-29daad2d9fa9.png">


can we come compromise maybe? i remove html. i put in a console.log
<img width="307" alt="image" src="https://user-images.githubusercontent.com/58625258/215297161-44d32f74-6abe-45ab-af94-4655131aebe0.png">

FichteFoll commented on index.html:
```diff
@@ -1,110 +1,94 @@
+               </a>
+            </div>
+            <div class="right">
+               <a href="https://discord.sublimetext.io/">
```
```suggestion
               <a href="http://discord.sublimetext.io/">
```

https doesn't work with the redirect, unfortunately.

aloofbit replied:
oops! i updated, sorry about that!

aloofbit committed:
fix url

aloofbit committed:
try improve code, remove heart

TerminalFi commented:
> remove github and put in .nojekyl. can host off main branch :root now

I’d prefer to keep everything hosted of off the gh-pages branch

aloofbit committed:
add gh_pages branch

aloofbit commented:
> > remove github and put in .nojekyl. can host off main branch :root now
> 
> I’d prefer to keep everything hosted of off the gh-pages branch

Ok ok! I'm not very familiar with the right buttons to press for that. I put the gh_pages.yaml file back can you help me on the proper settings?

aloofbit committed:
change to in

FichteFoll commented on src/script.js:
```diff
@@ -0,0 +1,39 @@
+
+function initialize(state) {
+   console.log('Made with ❤️ in Sublime Text')
+   const toggle = state === UI_DARK ? true : false
```
` const toggle = state === UI_DARK`

(github didn't allow me to make a suggestion for some reason)

aloofbit replied:
ooo that is tidy 💪 i updated!

aloofbit committed:
nice fix!

FichteFoll committed:
Adjust card spacing

FichteFoll committed:
Adjust min-height of container

Otherwise it wouldn't span the full height, though now it resizes only
lazily, but if only `height` was used it would be too small
sometimes …

FichteFoll commented:
Looks good to me. I made some small adjustments. 

Pipeline should also work as-is.
